schema/Mutation.graphql

```diff
@@ -84,4 +84,16 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create a new Life object
+    """
+    createlife(
+        firstName: String!, 
+        lastName: String!,
+        fullName: String!,
+        description: String!,
+        title: String!
+        birthday: DateTime!,
+        hobbies: [String!]!): Life!
```

Comment:
This is ok, but what you can do to improve this is to create an `Input` type for the arguments.

Reply:
Also, you do not need `fullname` as an input. As mentioned in the SK, `fullname` is just based on `firstName` and `lastName`

Reply:
Noted! Will look into improving the arguments with the `Input` type. I have removed the `fullName` field from the mutation.

Reply:
It's a cleaner way to read + you can easily understand the purpose because it is wrapped in an object

Reply:
Thank you! I have changed to the `input` type for the mutation.

The `input` type is defined in the `Life` type in `starter-kit/schema/types/Life.graphql`. May I ask if this is good practice? Or would it be better to create a new file to define the `input` type